App.tsx

```diff
@@ -255,6 +256,12 @@ class App extends React.Component<Props, any> {
       });
     }
 
+    // Kochava init
+    logBreadcrumb('App.js', `Kochava: initialize`);
+    KochavaTracker.instance.registerIosAppGuid(getEnv().KOCHAVA_IOS_ID);
+    KochavaTracker.instance.registerAndroidAppGuid(getEnv().KOCHAVA_ANDROID_ID);
```

Comment:
Just wondering if we should be checking whether this is on iOS or Android then calling either `registerIosAppGuid` or `registerAndroidAppGuid` accordingly? I don't know the behaviour but i'm worried it would send two events for the iOS and Android project regardless of platform.

Reply:
Yeah I just followed the example they had in their docs (which invoked both calls regardless of platform), but i can change that